web-app/src/hoc/Layout/header.jsx

```diff
@@ -1,55 +1,151 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import Swal from 'sweetalert2';
 import { PropTypes as PT } from 'prop-types';
 import { userLogout } from '../../utilities/currentUser';
-import roles from '../../utilities/roles';
-import { HeaderContainer, HeaderContent, HeaderItem } from './styled';
+//import roles from '../../utilities/roles';
+//import Legend from '../../components/Legend';
+//import UserDetails from '../../components/UserDetails';
+import { NavLink } from 'react-router-dom';
+import {
+  HeaderContent,
+  HeaderItem,
+  SubSection,
+  ToggleDrawer,
+  Icon,
+  Tooltip
+} from './styled';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import {
+  faUser,
+  faBars,
+  faTachometerAlt,
+  faUnlockAlt,
+  faSignOutAlt
+} from '@fortawesome/fontawesome-free-solid';
 
-const Header = ({ userDetails, history }) => {
+const Header = props => {
   const handleLogout = () => {
     Swal({
       title: 'Are you sure?',
       text: 'Do you wish to log out?',
       buttons: true,
       dangerMode: true,
-      showCancelButton: true,
+      showCancelButton: true
```

Comment:
Set prettier up with this setting in VSCode to avoid it changing these:
`"prettier.trailingComma": "all",`